api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+{
+        "swagger": "2.0",
+        "info": {
+                "title": "Cilium API",
+                "description": "Cilium",
+                "version": "unversioned"
+        },
+        "schemes": [
+                "http"
+	],
+        "x-schemes": [
+		"unix"
+        ],
+        "basePath": "/v1",
+        "produces": [
+                "application/json"
+        ],
+        "consumes": [
+                "application/json"
+        ],
+        "paths": {
+                "/healthz": {
+                        "get": {
+                                "summary": "Get health of Cilium daemon",
+                                "description": "Represents the runtime status of the daemon and related components.",
+				"tags": [ "daemon" ],
+                                "responses": {
+                                        "200": {
+                                                "description": "Success",
+                                                "schema": {
+                                                        "$ref": "#/definitions/StatusResponse"
+                                                }
+                                        }
+                                }
+                        }
+                },
+                "/config": {
```

Comment:
I guess it would depend on whether we think in future there could be similar paths for other cilium contexts where we would potentially run into collision. I'm fine either way, was just a thought.